gravity-forms/gw-time-sensitive-choices.php

```diff
@@ -27,9 +29,10 @@ public function __construct( $args = array() ) {
 		// set our default arguments, parse against the provided arguments, and store for use throughout the class
 		$this->_args = wp_parse_args( $args, array(
 			'form_id'       => false,
-			'field_ids'     => array(),
+			'field_id'      => false,
 			'time_mod'      => false,
```

Comment:
I think we should remove `time_mod` from the parameters since it's no longer used by the snippet.

Reply:
Agreed. And done!